src/deluge/processing/engines/audio_engine.h

```diff
@@ -203,4 +203,5 @@ extern Metronome metronome;
 extern RMSFeedbackCompressor mastercompressor;
 extern uint32_t timeLastSideChainHit;
 extern int32_t sizeLastSideChainHit;
+extern float rmsLevel;
```

Comment:
shouldn't be extern here - it should just be a global variable in this file, extern where you want to use it. It's probably getting declared implicitly right now